### PR TITLE
Built jars

### PR DESCRIPTION
**Pull request by fbertsch**
@jasonthomas tagged you for review as a follow-up to https://github.com/mozilla/telemetry-streaming/pull/119. Feel free to redirect as needed.

### CODE REVIEW

**Comment by fbertsch** [comment]
@whd updated with the changes from https://github.com/mozilla/telemetry-streaming/pull/121

File: jobs/focus_android_events_to_amplitude.sh [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+if [[ -z "$date" || -z "$max_requests" || -z "$key_file" || -z "$artifact" ]]; then
     echo "Missing arguments!" 1>&2
     exit 1
 fi
 
-git clone https://github.com/mozilla/telemetry-streaming.git
-cd telemetry-streaming
-sbt assembly
+artifact_filename="artifact.jar"
+aws s3 cp "$artifact" "$artifact_filename"
```
Comment (whd) [inline jobs/focus_android_events_to_amplitude.sh]: I'm annoyed that this is still an issue with EMR (client mode can't use s3 URLs directly).

File: Dockerfile.dev [hunk 2]
```diff
@@ -52,6 +54,8 @@ RUN mkdir -p /usr/share/man/man1 /usr/share/man/man7 && \
 COPY requirements.txt /tmp/
 # Switch to /tmp to install dependencies outside home dir
 WORKDIR /tmp
+
+RUN pip install pip==9.0.3
```
Comment (whd) [inline Dockerfile.dev]: (This is an unrelated but good change)

**Comment by whd** [comment]
@fbertsch I've redeployed WTMO but we need an additional change to this PR or emr-bootstrap-spark. Since these are public artifacts they can be requested via https at e.g. https://s3-us-west-2.amazonaws.com/net-mozaws-data-us-west-2-ops-ci-artifacts/mozilla/telemetry-batch-view/tags/telemetry-batch-view.jar, but to access them via the aws s3 api requires us to update our s3 permissions in the IAM role for emr-bootstrap-spark. I'm in favor of using the public URLs since these are public artifacts.

Also looks like this needs a rebase.

**Comment by whd** [comment]
:+1:

**Pull request merged**